news/2019-05-21-osu-mapping-olympiad-5-results.md

```diff
@@ -0,0 +1,52 @@
+| #9   | Innovation           | Comfortable Emu        | 24                     | 33              | 27             | 26              | 6                | **116**   |
+| #10  | [ Zzz ]              | Defensive Sapsucker    | 20                     | 26.5            | 21.5           | 21              | 10.5             | **99.5**  |
+| #11  | _fluffyy             | Apathetic Elephant     | 18.5                   | 26.5            | 15.5           | 16.5            | 7                | **84**    |
+| #12  | AlienSlayerX         | Unique Warbler         | 9                      | 14              | 8              | 9.5             | 3.5              | **44**    |
+
+In **first place** are [**Necho**](https://osu.ppy.sh/users/4086593) and [**Ryuusei Aika**](https://osu.ppy.sh/users/7777875) with a total score of **168**! All of the judges were impressed by their entry, which demonstrated excellence across all categories including achieving the highest scores for Musical Relevance and Impression. Congratulations on joining the ranks of osu!'s *Elite Mappers*!
+
+Creating both entries alone, in **second place** is [**yaspo**](https://osu.ppy.sh/users/4945926) with a total score of **161.5**! With the highest score across all entries for Creativity and Hitsounding, yaspo created a pair of excellent maps that were the highest scoring from judges Electoz and Myxomatosis.
+
+And in **third place** are [**J1_**](https://osu.ppy.sh/users/5918561) and [**how2miss**](https://osu.ppy.sh/users/4477199) with a total score of **157**! Alongside tying the winning entry for highest score in Technique, they were awarded the highest score from judge pishifat. All in all an excellent entry from this pair!
+  
+Thank you to everybody who submitted an entry and congratulations to all of the top three finishers! Here is what each person will receive as their reward (as a reminder, 6 Olympiad Points are needed for one badge level):
+
+- **First place**: 4 months of osu!supporter, 6 Olympiad Points ('Mapping Olympian' badge), *Elite Mapper*
+- **Second place**: 3 months of osu!supporter, 4 Olympiad Points
+- **Third place**: 2 month of osu!supporter, 2 Olympiad Points
+
+As a bonus for winning first place, each of the winners' badge descriptions will include the song they placed first in!
+
+[Here is a listing of all accumulated Olympiad Points](https://docs.google.com/spreadsheets/d/1_gIDJwTOgIhGr2h4069-r1C-2GHuV--5wtwPywzYsz8/edit?usp=sharing) which is updated after each contest.
+
+We'd like to thank all of the judges who spent their time going through and judging all of the entries: [Electoz](https://osu.ppy.sh/users/6485263), [Myxomatosis](https://osu.ppy.sh/users/2202645), [pishifat](https://osu.ppy.sh/users/3178418), and [Sonnyc](https://osu.ppy.sh/users/11771). Contests would not be able to be run without the time and effort of judges, and for that we are truly grateful for your efforts.
```

Comment:
```suggestion
We'd like to thank all of the judges who spent their time going through and judging all of the entries: [Electoz](https://osu.ppy.sh/users/6485263), [Myxomatosis](https://osu.ppy.sh/users/2202645), [pishifat](https://osu.ppy.sh/users/3178418), and [Sonnyc](https://osu.ppy.sh/users/11771). Contests would not be able to be run without the time and effort of judges, and for that we are truly grateful for their efforts.
```

Last sentence says "your efforts" (referring to the reader?), shouldn't it say "their efforts"?